src/phx/api/phx_api.py

```diff
@@ -79,6 +79,7 @@ def __init__(
         self.config: dict = config or {}
         self.mkt_symbols: Set[Ticker] = set([(exchange, symbol) for symbol in mkt_symbols])
         self.trading_symbols: Set[Ticker] = set([(exchange, symbol) for symbol in trading_symbols])
+        self.set_of_symbol_names = {symbol[1] for symbol in self.mkt_symbols.union(self.trading_symbols)}
```

Comment:
`self.mkt_symbols` and `self.trading_symbols` are list of symbols, not list of tuples.
this line should be
self.set_of_symbol_names = {symbol for symbol in self.mkt_symbols.union(self.trading_symbols)}

Reply:
self.mkt_symbols.union(self.trading_symbols) is a set of tickers: (exchange, symbol). So symbol[1] only fetches the symbol component.

Reply:
Mhy comment was wrong , the current implementation is correct.